src/config/routes/ErrorBoundary.tsx

```diff
@@ -0,0 +1,71 @@
+import { DateTime } from 'luxon'
+import { useEffect } from 'react'
+import { useRouteError } from 'react-router-dom'
+
+import { useServiceWorkerUpdate } from '../../context'
+import { NotFoundPage } from '../../pages/fallback'
+
+const FAILED_FETCH_ERROR = [
+  'Failed to fetch dynamically imported module',
+  'Importing a module script failed',
+  'error loading dynamically imported module',
+  'dynamically imported module',
+]
+const CHUNK_LOAD_ERROR = 'ChunkLoadError'
+const LOCAL_STORAGE_LAST_REFRESH_KEY = 'ChunkLoadError'
+const ONE_MINUTE_IN_MILIS = 60 * 1000
```

Comment:
Typo, should be `ONE_MINUTE_IN_MILLIS`, or `ONE_MINUTE_IN_MS` which is a bit more common from what I've seen.